docs/src/gatsby-theme-docz/index.tsx

```diff
@@ -34,6 +34,10 @@ interface Props {
 const docLayout = css`
   display: flex;
   flex-direction: row;
+  .docContainer {
+    word-wrap: break-all;
+    width: 100%;
```

Comment:
Yep, this fixes the problem. I'll undo a flexbox change I was messing with, to the same effect, on another PR.

Note though, that with this div being 100%, and the `.anchorNavStyle` div being `position: fixed`, I'm pretty darn sure that the `docLayout`'s `display: flex` and `flex-direction: row` styles have no effect/purpose.